packages/commonwealth/server/routes/getThreadCommunity.ts

```diff
@@ -0,0 +1,22 @@
+import { ServerError } from 'common-common/src/errors';
+import type { Request, Response } from 'express';
+import { Op } from 'sequelize';
+import type { DB } from '../models';
+
+const getThreads = async (models: DB, req: Request, res: Response) => {
```

Comment:
In general, I really dislike this approach of "we need some data about X? let's add a new route _purpose built_ for the particular use case."

Instead we should think of how we can make our existing API more clear + flexible + better designed to support a broad & general set of use cases.

Reply:
Feel free to update this PR with it. Reviewing bugs with BD, the issue here never got surfaced, so I'm unclear whether this PR even needs to go in / whether the ticket needs to be fixed. However, would love to see what you've done, because it can be cleanup work.

Reply:
Pushed